azure-pipelines.yml

```
@@ -86,10 +86,7 @@ jobs:
       steps:
       - checkout: self
         clean: true
-      - script: eng\common\cibuild.cmd
-          -configuration $(_BuildConfig)
-          -prepareMachine
-          $(_BuildArgs)
+      - powershell: eng\azpbuild.ps1 -configuration $(_BuildConfig) $(_BuildArgs)
```

It's not at all clear to me why `$(_BuildArgs)` isn't getting expanded by Pipelines and is instead passed all the way down to powershell. Maybe try the long form with explicit `arguments`?

```suggestion
      - task: PowerShell@2
        inputs:
          filePath: eng\azpbuild.ps1
          arguments: -configuration $(_BuildConfig) $(_BuildArgs)
```

---

I think it's because there is a combination in the `matrix` that doesn't define that variable. Just added something that might resolve that.

---

Nice! Now you're just up against the "pipelines doesn't plumb the errors through to GitHub Checks" problem, looks like--the summary page looks nice.